heartbeat/mysql

```diff
@@ -411,9 +411,9 @@ get_read_only() {
     local read_only_state
 
     read_only_state=`$MYSQL $MYSQL_OPTIONS_REPL \
-        -e "SHOW VARIABLES" | grep read_only | awk '{print $2}'`
+        -ss -e "SHOW GLOBAL VARIABLES LIKE 'read_only'"
 
-    if [ "$read_only_state" = "ON" ]; then
+    if [ $read_only_state =~ '*ON' ]; then
```

Comment:
On Mon, Feb 22, 2016 at 07:14:15AM -0800, Kristoffer Grönlund wrote:
> >  
> > -    if [ "$read_only_state" = "ON" ]; then
> > +    if [ $read_only_state =~ '*ON' ]; then
> 
> `=~` is not in POSIX test either

Right, so it shouldn't be used. Just wanted to warn about
introducing stuff which aren't available in /bin/sh.